README.rst

```diff
@@ -52,17 +52,34 @@
 Welcome to sktime
 =================
 
-  A unified framework for machine learning with time series
+The unified framework for machine learning with time series. Our vision:
 
-We provide specialized time series algorithms and `scikit-learn
-<https://github.com/scikit-learn/scikit-learn>`__ compatible tools
-to build, tune and validate time series models for multiple learning problems, including:
+* **by the community**, for the community. Developed by a friendly, decentral community - `we welcome all contributors <https://gitter.im/sktime/community>`__.
```

Comment:
To me, having the vision at the top of the readme is a bit long, it also reads a bit too opinionated for a landing page I think. 

As an example, I really like spaces readme: https://github.com/explosion/spaCy/blob/master/README.md Short description, then well-organised overview tables with important links, then short summaries of features, installation, etc.

Reply:
I prefer the vision statement to go into the community roadmap

Reply:
hm, I prefer it here - @TonyBagnall, what do you think?